### DISCUSSION TITLE
Successful demos

### DISCUSSION THREAD

**Discussion by jiweiqi** [jiweiqi/CellBox.jl#12] [Show and tell]
```yaml
is_restart: true
ns: 20

tfinal: 8.0
ntotal: 20
batch_size: 8

lr: 1.e-3
weight_decay: 1.e-6

n_mu: 50

n_exp_train: 20
n_exp_val: 5
n_exp_test: 5
noise: 0.01

n_iter_max: 10000
n_plot: 10 # frequency of callback

n_iter_buffer: 500
n_iter_burnin: 500
n_iter_tol: 500
convergence_tol: 1e-8

sparsity: 0.95
drop_range:
   lb: -0.1
   ub: 0.1

```

![image](https://user-images.githubusercontent.com/8445647/107451317-078b7f80-6b15-11eb-9216-e1c69be7e01f.png)
![image](https://user-images.githubusercontent.com/8445647/107451329-0ce8ca00-6b15-11eb-8213-04e1b952553c.png)
A testing data
![image](https://user-images.githubusercontent.com/8445647/107451346-13774180-6b15-11eb-9bbd-0ac426d04a26.png)

**Comment by jiweiqi** [comment]
* the weight_decay seems not necessary and might be negative .... Instead, we can use the data to regularize.
* continual learning seems to be useful. It proceeds by training with a small number of conditions, say 5. After overfitting those 5 samples, we increase the number of conditions.

**Comment by jiweiqi** [comment]
It is useful to look into the parameter correlation to determine the progress of training, e.g., This one indicates that the training is still going on
![image](https://user-images.githubusercontent.com/8445647/107461341-ca7cb880-6b27-11eb-8df4-a064a128fe3b.png)

**Comment by jiweiqi** [comment]
```yaml
is_restart: true
ns: 50

tfinal: 8.0
ntotal: 20
batch_size: 16

lr: 1.e-3
weight_decay: 1.e-6

n_mu: 50

n_exp_train: 40
n_exp_val: 5
n_exp_test: 5
noise: 0.01

n_iter_max: 10000
n_plot: 10 # frequency of callback

n_iter_buffer: 500
n_iter_burnin: 500
n_iter_tol: 50000
convergence_tol: 1e-8

sparsity: 0.95
drop_range:
   lb: -0.1
   ub: 0.1
```

![image](https://user-images.githubusercontent.com/8445647/107585715-b939b780-6bcc-11eb-97a1-142b570830ed.png)
![image](https://user-images.githubusercontent.com/8445647/107585745-c35bb600-6bcc-11eb-95a5-c193fe8a9d62.png)

The loss is still decreasing

**Reply by jiweiqi** [reply]
employed learning rate decaying and continual training
